src/components/DirectAnswer.tsx

```diff
@@ -39,11 +40,7 @@ export interface DirectAnswerCssClasses extends ThumbsFeedbackCssClasses {
   viewDetailsLink?: string,
   viewDetailsLinkContainer?: string,
   highlighted?: string,
-  answerContainer?: string,
-  feedbackButtonsContainer?: string,
-  feedbackButton?: string,
-  thumbsUpIcon?: string,
-  thumbsDownIcon?: string
```

Comment:
I believe these were here because its a little nicer for the docs to display the full list of classes rather than having to click through to see the interface which is being extended. However 'feedbackButton' doesn't look like it's being used anywhere so we can remove that one regardless

Reply:
I see. I think it might still be better to remove these for consistency though, because the other interfaces don't include the full list of classes when they extend other classes.
If we want to always display the full list of classes, that might be better to do in a separate item